DesktopReact/frontend/src/components/userProfileModal/UserProfileAvatarModal.js

```diff
@@ -1,27 +1,81 @@
-import React, {useState} from 'react';
+import React from 'react';
 import AvatarEditor from "react-avatar-editor";
-import ReactAvatarEditor from 'react-avatar-editor';
-import {Modal} from "@mui/material";
+import {Box, Button, IconButton, Modal, Typography} from "@mui/material";
+import {CloseOutlined, ZoomInOutlined, ZoomOutOutlined} from "@mui/icons-material";
 
-const UserProfileAvatarModal = (props) => {
+const UserProfileAvatarModal = ({ modalEditAvatar,
+                                    modalEditBanner,
```

Comment:
This 2 variables could be a single enum with 2 values and then you check which value is the actual